projectile-rails.el

```diff
@@ -1685,7 +1690,12 @@ Killing the buffer will terminate to server's process."
   (add-hook 'compilation-filter-hook 'projectile-rails-server-compilation-filter)
   (add-hook 'kill-buffer-hook 'projectile-rails-server-terminate t t)
   (add-hook 'kill-emacs-hook 'projectile-rails-server-terminate t t)
-  (setq-local compilation-scroll-output t)
+
+  (require 'comint)
+  (add-hook 'compilation-filter-hook 'comint-truncate-buffer)
```

Comment:
This modifies the hook globally, for all compilations, which I think is not what is intended here.